packages/document-service-api/src/controllers/documents.js

```diff
@@ -77,10 +77,12 @@ module.exports = {
         data: fileData,
       });
     } else {
+      const fileName = doc.get('name');
       const mimeType = doc.get('mimeType');
       req.log.info({ mimeType }, 'Displaying file');
 
       res.set('content-type', mimeType);
+      res.set('x-original-file-name', fileName);
```

Comment:
pass the original file name through as a header, as this will be useful for 'naming' the file that will be downloaded from FWA